### DISCUSSION TITLE
[프로그래밍 언어론]  Programing Language - 프로그래밍 언어의 발전사: 언어를 선택하지 말라

### DISCUSSION THREAD

**Discussion by 02ggang9** [BDD-CLUB/challenge-every-week-posting#450] [1. 개발 관련 글 모음]
[Programing Language - 프로그래밍 언어의 발전사: 언어를 선택하지 말라](https://02ggang9.github.io/etc/pl1/)

허허,, 시험기간에 글을 쓰는게 맞는지 모르겠네요..

**Comment by shkisme** [comment]
오... 잘 읽었습니다 수업 내용이 앞으로의 공부 방향성 잡는데 많은 도움이 될 것 같아 보이네요... 👍
